_search-plugins/star-tree-index.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: default
+title: Star-tree index
+parent: Improving search performance
+nav_order: 54
+---
+
+# Star-tree index
+
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress the feature or if you want to leave feedback, join the discussion on the [OpenSearch forum](https://forum.opensearch.org/).    
+{: .warning}
+
+Star-tree index is a multi-field index that improves the performance of aggregations.
+
+OpenSearch will automatically use the star-tree index to optimize aggregations if the fields queried are part of star-tree index dimension fields and the aggregations are on star-tree index metrics fields. No changes are required in the query syntax or the request parameters.
+
+## When to use a star-tree index
+
+Star-tree index can be used to perform faster aggregations. Consider the following criteria and features when deciding to use a star-tree index:
+
+- Star-tree indexes natively support multi-field aggregations.
+- Star-tree indexes will be created in real-time as part of the indexing process, so the data in a star-tree will always be up to date.
+- A star-tree index consolidates data making the index efficient at paging and using less IO for search queries.
+
+## Limitations
+
+Star-tree indexes contain the following limitations:
+
+- Star-tree index should only be used on indexes whose data is not updated or deleted, as updates and deletions are not accounted for in a  star-tree index.
```

Comment:
```suggestion
- A star-tree index should only be used on indexes whose data is not updated or deleted because updates and deletions are not accounted for in a star-tree index.
```